x/payment/types/events.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	EventTypePayForMessage = "payformessage"
+
+	AttributeKeySpender = "spender"
+	AttributeKeySize    = "size"
+)
+
+//NewPayForMessageEvent construt a new payformessge sdk.Event
+func NewPayforMessageEvent(spender sdk.AccAddress, size uint64) sdk.Event {
+	return sdk.NewEvent(
+		EventTypePayForMessage,
+		sdk.NewAttribute(AttributeKeySpender, spender.String()),
+		sdk.NewAttribute(AttributeKeySize, string(size)),
```

Comment:
per the linter, we will want to convert the uint64 to a string via some other mechanism such as `[FormatUint](https://pkg.go.dev/strconv#FormatUint)`

Reply:
I changed to `FormatUint(size, 10)`